metadata-ingestion/tests/integration/delta_lake/golden_files/local/golden_mces_tables_with_nested_datatypes.json

```diff
@@ -96,11 +96,11 @@
                                 "jsonProps": "{\"native_data_type\": \"struct<_1:long,_2:string>\"}"
                             },
                             {
-                                "fieldPath": "[version=2.0].[type=struct].[type=struct].data.[type=struct]._2.[type=null]._1",
+                                "fieldPath": "[version=2.0].[type=struct].[type=struct].data.[type=struct]._2.[type=long]._1",
                                 "nullable": true,
                                 "type": {
                                     "type": {
-                                        "com.linkedin.pegasus2avro.schema.NullType": {}
+                                        "com.linkedin.pegasus2avro.schema.NumberType": {}
```

Comment:
Note for reviewers:
This is for the new mapping I added here 
https://github.com/datahub-project/datahub/blob/21b3c4773f3aac54767cd5291d6d249bc88a1346/metadata-ingestion/src/datahub/utilities/hive_schema_to_avro.py#L31